tests/IntegrationTest/Spoonacular/Recommendation/HomeTest.test.jsx

```diff
@@ -0,0 +1,173 @@
+import { describe, test, expect } from "vitest";
+import { render, screen, waitFor } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import HomeTest from "src/pages/Home/HomeTest";
+import { BrowserRouter } from "react-router-dom";
+import axios from "axios";
+
```

Comment:
when i run it, if i import env, i dont get the env not found error anymore. maybe can you try that?